Makefile

```diff
@@ -244,3 +244,14 @@ check_clean: check_env
 .PHONY: openapi
 openapi:
 	python $(project_root)/scripts/generate_openapi_document.py
+
+aws_provider_count = $(shell git secrets --list | grep -c -e '--aws-provider')
+
+.PHONY: check_git_secrets
+check_git_secrets:
+ifeq ($(aws_provider_count),1)
```

Comment:
Still wrong, by the way. `ifeq` is evaluated when the Makefile is parsed, not when the target is invoked. Please provide evidence if you think I am wrong. We'll remove the target but do this anyways.

Reply:
Sorry, I fail to see why the time of evaluation makes a difference in this case. The output from `git secrets` should be the same when the Makefile is parsed and when the target is invoked.

The following console output shows how the make target worked as intended<sup>1</sup>. Since I couldn't (easily) uninstall git-secrets from my system I modified the grep command instead.

<sup>1</sup>Side note, I realize now that this approach (with `git secrets --list`) is an inadequate test since it only confirms that git-secrets is installed and the AWS patterns have been configured, however it does not confirm the hooks have been added into the repository.

```
(.venv) daniel@Crispin ~/repo/azul1 $ tail -n 11 Makefile

aws_provider_count = $(shell git secrets --list | grep -c -e '--aws-provider')

.PHONY: check_git_secrets
check_git_secrets:
ifeq ($(aws_provider_count),1)
	@echo "git-secrets check... PASSED.";
else
	@echo "git-secrets check... FAILED."
	@false
endif
(.venv) daniel@Crispin ~/repo/azul1 $ make check_git_secrets
git-secrets check... PASSED.
```

```
(.venv) daniel@Crispin ~/repo/azul1 $ vim Makefile
(.venv) daniel@Crispin ~/repo/azul1 $ tail -n 11 Makefile

aws_provider_count = $(shell git secrets --list | grep -c -e '--aws-provider_x')

.PHONY: check_git_secrets
check_git_secrets:
ifeq ($(aws_provider_count),1)
	@echo "git-secrets check... PASSED.";
else
	@echo "git-secrets check... FAILED."
	@false
endif
(.venv) daniel@Crispin ~/repo/azul1 $ make check_git_secrets
git-secrets check... FAILED.
make: *** [check_git_secrets] Error 1
```